bears/c_languages/CSecurityBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+@linter(executable="flawfinder",
+        output_format="regex",
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):\s*'
+                     r'\[(?P<severity>\d)\]\s*'
+                     r'\((?P<origin>.+)\) (?P<message>.+)',
+        severity_map={"1": RESULT_SEVERITY.INFO,
+                      "2": RESULT_SEVERITY.INFO,
+                      "3": RESULT_SEVERITY.NORMAL,
+                      "4": RESULT_SEVERITY.NORMAL,
+                      "5": RESULT_SEVERITY.MAJOR})
+class CSecurityBear:
+    """
+    Report possible security weaknesses for C/C++.
+
+    For more information, consult <http://www.dwheeler.com/flawfinder/>.
+    """
+
+    LANGUAGES = {'C', 'C++'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Security', 'Memory Leak', 'Code Simplification'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return "--columns", "--dataonly", "--quiet", "--singleline", filename
```

Comment:
all those options are added to make the output match with (output_regex) 
``columns``: dispaly the column  number as well as the line number and filename
``quiet``:  prevents flawfinder from displaying status infos 
``singleline``: singleline output
``dataonly``: use along with quiet